src/models/MediaMetadata.ts

```diff
@@ -8,18 +8,30 @@ mongoose.set('useCreateIndex', true);
 
 export interface MediaMetadataInterface extends Document {
   actors: Array<string>;
+  awards?: string;
+  boxoffice?: string;
+  country?: string;
   directors: Array<string>;
   episodeNumber?: string;
   episodeTitle?: string;
   genres: Array<string>;
   goofs?: string;
   imdbID: string;
+  metascore?: string;
   osdbHash?: string;
+  production?: string;
+  poster?: string;
+  rated?: string; // e.g 'PG-13'
+  rating?: number; // e.g. 6.7
+  ratings?: Array<{Source: string; Value: string}>; // e.g. {"Source": "Metacritic", "Value": "67/100"}
```

Comment:
It would be cool to have interfaces/enums for the sources instead of having the e.g.

Reply:
Do you think the three we know about are the only options?

Reply:
Actually I was picturing losing documents due to Mongoose validation, but this is just the types. I'll add the enums here later